node/src/chain/fsm.rs

```diff
@@ -360,17 +384,23 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network>
             return Ok(false);
         }
 
+        let enable_consensus = true;
+
```

Comment:
Why this is fixed to `true`?

Reply:
As per `Enable consensus task when a block is accepted in OutOfSync state`